recipes/numcodecs/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.0.1" %}
+
+package:
+  name: numcodecs
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/alimanfoo/numcodecs.git
+  git_tag: v{{ version }}
```

Comment:
Yep. That is actually desirable so we ensure the same tarball is used. Github allow to remove and re-issue a version but with the sha256 we will know that it changed :smile: